### ISSUE TITLE
Helmet Error

### ISSUE THREAD

**Issue by Da-Colon** [decent-dao/wallet-provider#8]
![Image](https://user-images.githubusercontent.com/38386583/175450034-2a213a64-470c-4590-94f3-4be115167f4b.png)

An error occurs when the provider is switch out with the install

**Comment by Da-Colon** [comment]
Yeah so I was wondering why the build was so large on npm. testing the the error was made during publishing the package. I will publish and test the next with next release.

**Comment by Da-Colon** [comment]
So it all came down to how the files were being complied and the other blocker was how `npm link` works when testing locally. 

When testing locally different versions of react render (react-dom in this case) would try to render the package (on one version) and the local app being ran in another, which caused  it throw different errors at times,  'Can't use hooks outside of React components'.

When testing a published package is installed error would be the above. 

After some testing. I've updated the `peerDependencies` property to 

```json
  "peerDependencies": {
    "react": "^16 || ^17 || ^18",
    "react-dom": "^16 || ^17 || ^18"
  },
```

also re distributed the the dependencies within package.json (I had misinterpreted the documentation on how to configure dependencies for `microbundle`).
